ext/scripts/list_newest_versions/list_candidate_versions_for_apt.sh

```diff
@@ -4,11 +4,13 @@ set -e
 set -u
 set -o pipefail
 
-PACKAGE_LIST_FILE=$1
+PACKAGE_LIST_FILE="$1"
 SCRIPT_DIR="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
 
-PACKAGE_LIST=( $($SCRIPT_DIR/extract_columns_from_package_lisl.pl --file $PACKAGE_LIST_FILE --columns 0) )
-VERSION_LIST=( $($SCRIPT_DIR/extract_columns_from_package_lisl.pl --file $PACKAGE_LIST_FILE --columns 1) )
+#shellcheck disable=SC2207
```

Comment:
https://github-wiki-see.page/m/koalaman/shellcheck/wiki/SC2207
Can be ignored

Reply:
yes, the perl scripts take core for the correct formatting